### PR TITLE
Group Fixes

### PR DESCRIPTION
**Pull request by dfreerksen**
Groups stopped being created and users stopped being added to groups. This fixes that. Plus the added bonus of a _little_ bit of documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,97 @@
...
+
+Copy the `Resources/examples/config.dist.php` file as `config.php`
+
+```
+cp Resources/examples/config.dist.php Resources/examples/config.php
+```
+
+Insert the correct value for `base_url`, `api_key` and `sso_secret`. The value for `api_username` should stay as `system`
+
+> NOTE: A `.gitignore` file in in place to prevent you from committing this file.
```
Comment (stephenp) [inline README.md]: is* in place?

**Comment by vinniefranco** [comment]
Looks good. Guzzle blows.

**Comment by vinniefranco** [comment]
:shipit:

**Comment by mcrumm** [comment]
@vinniefranco Maybe I just haven't had enough Haterade today, but I fail to see how this has anything to do with Guzzle.

Regardless, this looks good @dfreerksen.

**Comment by vinniefranco** [comment]
@mcrumm sorry, just echoing the find and debug frustration @dfreerksen was encountering.

**Comment by vinniefranco** [comment]
@dfreerksen could you bump the version here: https://github.com/groundctrl/discourse-api-php/blob/master/src/Discourse.php#L11

**Comment by mcrumm** [comment]
@vinniefranco I felt the same way, digging through the https://github.com/discourse/discourse_api commits, trying to figure out what the hell they changed. :disappointed:

**Comment by vinniefranco** [comment]
@mcrumm agreed. JERKS!

File: README.md [hunk 2]
```diff
@@ -1,10 +1,101 @@
...
+Using [composer](https://packagist.org/packages/ctrl/discourse):
 
-```bash
-$ composer require ctrl/discourse-api
+```
+composer require ctrl/discourse
+```
+
+```
+composer install
```
Comment (mcrumm) [inline README.md]: Nitpicky, to be sure, but you don't *need* the `composer install` line.  `compose require` will get the job done.

**Comment by vinniefranco** [comment]
Awesome! @dfreerksen squash those commits and :ship: :it:

**Comment by dfreerksen** [comment]
Ha! I just got that. ship + Italy flag. Ship it. Clever.

**Pull request merged**